lib/potassium/recipes/rack_cors.rb

```diff
@@ -16,10 +16,11 @@ def rack_cors_config
       config.middleware.insert_before 0, Rack::Cors do
         allow do
           origins '*'
-          resource '*',
+          resource '/public/*', headers: :any, methods: :get
+          resource '/api/*',
             headers: :any,
             expose: ['X-Page', 'X-PageTotal'],
-            methods: [:get, :post, :delete, :put, :options]
+            methods: [:get, :post, :patch, :put, :options]
```

Comment:
por qué sacaste el `delete` @rjherrera? Estamos dejando de usarlo?

Reply:
mi idea era que como es _destructivo_ fuera un opt-in, en otras partes vi que para las api solo dejan `:get` y `:post`, y pensé ser más permisivo y dejar usar patch y post, pero quizás hay que dejar todos los métodos y chao, no se qué opinai